solace-spring-boot-parent/pom.xml

```diff
@@ -21,15 +21,17 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
 
-        <solace.spring.boot.java-autoconf.version>4.3.1-SNAPSHOT</solace.spring.boot.java-autoconf.version>
-        <solace.spring.boot.jms-autoconf.version>4.3.1-SNAPSHOT</solace.spring.boot.jms-autoconf.version>
+        <solace.spring.boot.java-autoconf.version>5.0.0-SNAPSHOT</solace.spring.boot.java-autoconf.version>
+        <solace.spring.boot.jms-autoconf.version>5.0.0-SNAPSHOT</solace.spring.boot.jms-autoconf.version>
 
-        <solace.jcsmp.version>10.17.0</solace.jcsmp.version>
-        <solace.jms.version>10.17.0</solace.jms.version>
+        <solace.jcsmp.version>10.19.1-jakarta_jms.43138</solace.jcsmp.version>
+        <solace.jms-jakarta.version>10.19.1-jakarta_jms.43138</solace.jms-jakarta.version>
```

Comment:
These don't seem to be official release versions: https://mvnrepository.com/artifact/com.solacesystems/sol-jcsmp